Submissions/@o-ponchik.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Mariia Matviian",
```

Comment:
For confirmation - is it Mariia or Maria?

Reply:
the correct one is Mariia.